haystack/schema.py

```diff
@@ -1001,19 +1007,26 @@ def wrong_examples(
             wrong_examples = []
             for multilabel_id, metrics in worst_df.iterrows():
                 query_answers = answers[answers["multilabel_id"] == multilabel_id]
-                query_dict = {
-                    "multilabel_id": query_answers["multilabel_id"].iloc[0],
-                    "query": query_answers["query"].iloc[0],
-                    "filters": query_answers["filters"].iloc[0],
-                    "metrics": metrics.to_dict(),
-                    "answers": query_answers.drop(
-                        ["node", "query", "type", "gold_answers", "gold_offsets_in_documents", "gold_document_ids"],
-                        axis=1,
-                    ).to_dict(orient="records"),
-                    "gold_answers": query_answers["gold_answers"].iloc[0],
-                    "gold_document_ids": query_answers["gold_document_ids"].iloc[0],
-                }
-                wrong_examples.append(query_dict)
+                if answer_metric not in metrics:
+                    logger.warning(
+                        f"You specified an answer_metric={answer_metric} not available in calculated metrics={metrics.keys()}."
+                        f"Skipping collection of worst performing samples."
+                    )
+                    break
+                if metrics[answer_metric] >= answer_metric_threshold:
```

Comment:
`<=` instead of `>=` as we're looking for worst examples. The lower f1 the worse the example.